opsep: Transferring ownership of L2PA to aliased L1PAO address (#204)
ethereum-optimism/superchain-ops: tasks/opsep/001-update-l2pao-to-aliased-l1pao/README.md

# Op Sepolia - Update L2ProxyAdmin owner to be aliased L1ProxyAdmin owner address

> Status: [EXECUTED](https://sepolia-optimism.etherscan.io/tx/0x06953e9fa4f05c9c074749c0d702647a67348d44f83c43948a7c8437e581f790#eventlog)

Reviewer: ```suggestion
Status: [EXECUTED](https://sepolia-optimism.etherscan.io/tx/0x06953e9fa4f05c9c074749c0d702647a67348d44f83c43948a7c8437e581f790)
```

## Objective

> This is the playbook for executing Upgrade 001, to update the L2ProxyAdmin owner (L2PAO) to be the aliased L1ProxyAdmin owner (L1PAO) address, on OP Sepolia (chain ID 11155420).

Reviewer: [Upgrade 1](https://gov.optimism.io/t/final-upgrade-1-bedrock-protocol-upgrade-v2/5548) is bedrock, this one is part of [Upgrade 8](https://gov.optimism.io/t/final-protocol-upgrade-8-guardian-security-council-threshold-and-l2-proxyadmin-ownership-changes-for-stage-1-decentralization/8157/1)

```suggestion
This is the playbook for executing part of [Upgrade #8](https://gov.optimism.io/t/final-protocol-upgrade-8-guardian-security-council-threshold-and-l2-proxyadmin-ownership-changes-for-stage-1-decentralization/8157/1), to update the L2ProxyAdmin owner (L2PAO) to be the aliased L1ProxyAdmin owner (L1PAO) address, on OP Sepolia (chain ID 11155420).
```

Author: Do I then need to change the name of this directory from `001-update-l2pao-to-aliased-l1pao` to `008-update-l2pao-to-aliased-l1pao`?

Reviewer: Nope, but I can see how is a bit confusing and something we may want to clarify in the readme:
- The upgrade number being referred to is related to governance, this is the 8th governance proposal
- The 001 prefix on the folder is the ordering of tasks in the opsep director, so this is the first one

(I unresolved for visibility, feel free to resolve when you want)

> Before this task was created, some op-sepolia configuration was updated to closely mirror op-mainnet. 

Reviewer: naming nit
```suggestion
Before this task was created, some OP Sepolia configuration was updated to closely mirror OP Mainnet. 
```

> Specifically, the L2Proxy Admin owner on op-mainnet is a Safe: [0x7871d1187A97cbbE40710aC119AA3d412944e4Fe](https://optimistic.etherscan.io/address/0x7871d1187A97cbbE40710aC119AA3d412944e4Fe). On op-sepolia it was an EOA [0xfd1D2e729aE8eEe2E146c033bf4400fE75284301](https://sepolia-optimism.etherscan.io/address/0xfd1D2e729aE8eEe2E146c033bf4400fE75284301). It has since been updated to be a 1-of-1 safe: [0xb41890910b05dCba3d3dEF19B27E886C4Ab406EB](https://sepolia-optimism.etherscan.io/address/0xb41890910b05dCba3d3dEF19B27E886C4Ab406EB), where the previous L2Proxy Admin Owner EOA is it's signatory.

Reviewer: nit, Safe calls them "owner", we use "owner" and "signer" interchangably but I haven't seen signatory used before so just want to avoid confusion
```suggestion
Specifically, the L2Proxy Admin owner on op-mainnet is a Safe: [0x7871d1187A97cbbE40710aC119AA3d412944e4Fe](https://optimistic.etherscan.io/address/0x7871d1187A97cbbE40710aC119AA3d412944e4Fe). On op-sepolia it was an EOA [0xfd1D2e729aE8eEe2E146c033bf4400fE75284301](https://sepolia-optimism.etherscan.io/address/0xfd1D2e729aE8eEe2E146c033bf4400fE75284301). It has since been updated to be a 1-of-1 safe: [0xb41890910b05dCba3d3dEF19B27E886C4Ab406EB](https://sepolia-optimism.etherscan.io/address/0xb41890910b05dCba3d3dEF19B27E886C4Ab406EB), where the previous L2Proxy Admin Owner EOA is it's owner.
```
